uuid.go

```diff
@@ -182,22 +174,33 @@ func (u UUID) Bytes() []byte {
 	return u[:]
 }
 
+// encodeHex encodes the canonical RFC-4122 form of UUID u into the
+// first 36 bytes dst.
+func encodeHex(dst []byte, u UUID) {
```

Comment:
Maybe call this `encodeCanonical()` for accuracy?  When I read `encodeHex` I'm expecting the output to be a string of hex digits.

Reply:
Done